Assets/Scripts/UI/Views/Popups/Pause/PauseMenuObserver.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEngine;
+using Zenject;
+
+namespace UI
+{
+    public class PauseMenuObserver : MonoBehaviour
+    {
+        private IUIRoot _uiRoot;
+        private PauseMenu _pauseMenu;
+        private PauseManager _pauseManager;
+    
+        [Inject]
+        public void Construct(IUIRoot uiRoot)
+        {
+            _uiRoot = uiRoot;
+        }
+        private void Update()
+        {
+            if (Input.GetKeyDown(KeyCode.Escape))
+            {
+                if (_pauseMenu)
+                {
+                    Debug.Log("Pause menu observer: Pause menu closed");
```

Comment:
Remove debug

Reply:
We leave the output of debug information only if it is necessary for logging errors or important game states (scene loading, player death).

Reply:
Done